packages/tooltip/src/vaadin-tooltip-overlay.js

```diff
@@ -10,6 +10,10 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 registerStyles(
   'vaadin-tooltip-overlay',
   css`
+    [part='overlay'] {
+      max-width: 40ch;
```

Comment:
I'm probably really pedantic but should there be some upper bound as well to make sure it doesn't have the full width on mobile / smaller / zoomed view ports? Like using 90% as well like Rolf had in his original idea?

Reply:
Good catch! Thanks 🙇‍♂️